python/paddle/v2/fluid/distribute_transpiler.py

```diff
@@ -269,6 +271,7 @@ def _create_var_for_trainers(self, block, var, trainers):
                 name="%s.trainer_%d" % (var.name, i),
                 psersistable=var.persistable,
                 dtype=var.dtype,
+                type=var.type,
```

Comment:
Seems we are merging splited `SelectedRows` using `concat_op` and it's not supported?

Reply:
`sum_op` implement the same feature, so I reuse it.